### ISSUE TITLE
[Onboarding] Domain onboarding screen broken at larger dynamic text sizes

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-ios#14976]
**Describe the bug**
When the device is set to larger dynamic text sizes:

1. The primary button label is truncated.
2. The scrolling content can be seen below the primary button area.

**To Reproduce**
1. Set the device text to a larger size, e.g. 190%.
2. Log in to a Jetpack-connected store that hasn't completed the domain onboarding step (e.g. the Jetpack login test store for our internal smoke testing).
3. On the dashboard, tap on "View all 3 tasks" in the onboarding card.
4. In the onboarding task list, tap on "Customize your domain."
5. Notice the button label is truncated and the scrollable content can be viewed at the bottom of the screen, under the primary button overlay.

**Screenshots**

<img src="https://github.com/user-attachments/assets/9a50b623-8ea6-4f50-89a9-4ab0179e9d6d" width="300px">

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by itsmeichigo** [comment]
Closing because I cannot reproduce this issue:

<img src="https://github.com/user-attachments/assets/9cf2f9f5-8505-4584-a007-f0aa2b7dae08" width=320 />
